src/Spago/CLI.hs

```diff
@@ -193,16 +193,19 @@ parser = do
       , Repl <$> replPackageNames <*> sourcePaths <*> pursArgs <*> depsOnly
       )
 
+    firstNonEmpty x y = if null x then y else x
+    execArgs = firstNonEmpty <$> backendArgs <*> nodeArgs
+
     test =
       ( "test"
       , "Test the project with some module, default Test.Main"
-      , Test <$> mainModule <*> buildOptions <*> nodeArgs
+      , Test <$> mainModule <*> buildOptions <*> execArgs
```

Comment:
I'm now wondering if we should now call the new flag `--exec-args` or `--run-args` rather than `--backend-args`? :thinking:

Reply:
Yea, I like `exec-args` better. I guess my variable names betray that.